tfhe/docs/fundamentals/serialization.md

```diff
@@ -164,3 +164,6 @@ fn main() {
     ).is_ok());
 }
 ```
+
+This feature can also be combined with the [data versioning](../guides/data\_versioning.md) feature by using the `safe_serialize_versioned`
+and `safe_deserialize_conformant_versioned` functions.
```

Comment:
There's an extra line return in the sentence. 
Does this feature refer to the "serialization/deserialization" or the "safe serialization/deserialization"? Maybe it's better to specify it in the sentence.

Revised:
You can combine this serialization/deserialization feature with the [data versioning](../guides/data\_versioning.md) feature by using the `safe_serialize_versioned` and `safe_deserialize_conformant_versioned` functions.